source/interface.cc

```diff
@@ -28,10 +29,53 @@ Interface<dim,spacedim,n_components,LAC>::
 set_time (const double &t) const
 {
   dirichlet_bcs.set_time(t);
+  // dirichlet_dot.set_time(t);
```

Comment:
Because it is used in his work by Alberto.
This cose has not implemented *dirichlet_dot* yet.

> On 07 Nov 2015, at 19:12, Luca Heltai <notifications@github.com> wrote:
> 
> In source/interface.cc <https://github.com/mathLab/pi-DoMUS/pull/42#discussion_r44215143>:
> 
> > @@ -28,10 +29,53 @@ Interface<dim,spacedim,n_components,LAC>::
> >  set_time (const double &t) const
> >  {
> >    dirichlet_bcs.set_time(t);
> > +  // dirichlet_dot.set_time(t);
> Why is this commented?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mathLab/pi-DoMUS/pull/42/files#r44215143>.
>